### DISCUSSION TITLE
Http Output Plugin: multiple URI's or fallback URI should one fail

### DISCUSSION THREAD

**Discussion by cdancy** [fluent/fluent-bit#8108] [General]
Hey All,

Is there any support, either hidden or planned, that would allow the HTTP Output plugin to forward logs to some _other destination_ should things fail after multiple attempts? Would you all be interested in taking a PR for this functionality if none already exists?

Thanks,
Chris

**Comment by patrick-stephens** [comment]
There are some feature requests around this I believe if you have a look - and more general too, i.e. a  fallback output for any output rather than some special http output behaviour (plus http output is the basis for quite a few other custom plugins).

**Comment by cdancy** [comment]
@patrick-stephens  thanks. Any plans for doing something like this in the future or anything on the roadmap? If not we may take on work to do this internally and potentially send the PR upstream if you all wanted to take the work. Do you know of any branches/PR's of others doing so just so that we're not re-inventing the wheel?

**Reply by patrick-stephens** [reply]
I am not aware of anything specific but it is hard to keep track of everything on the project hence why I suggested finding the feature request for it that already exists and seeing what is discussed there. Of course, contribution would be lovely as it is all OSS anyway - just make sure to follow the contribution guidance.
